userprofile/views.py

```diff
@@ -0,0 +1,42 @@
+from django.shortcuts import render
+from rest_framework.decorators import api_view
+from rest_framework.response import Response 
+from django.contrib.auth import get_user_model
+from django.http import HttpResponse
+
+# Create your views here.
+User = get_user_model()
+
+def Home(request):
+    return HttpResponse("hello")
+
+@api_view(['POST'])
+def SignUp(request):
+    data = request.data
+    
+    if not data.__contains__('email') or not data.__contains__('password') or not data.__contains__('profile_id') or not data.__contains__('mobile') or not data.__contains__('name') or not data.__contains__('country'):
```

Comment:
Move all the keys(`email`, `password`, `profile_id`, etc) to the top of the file and make them in capital letters(To indicate constants). 
Example: https://github.com/DhrubojyotiBis1/Hexbit.io/blob/product/product/views.py